### ISSUE TITLE
Often pressing the download button does nothing

### ISSUE THREAD

**Issue by bbdbby** [tehrobber/civitai-one-click-dl#8]
I haven't quite figured out what is happening, but it could be related to the switch to the new image server over at Civitai... but roughly 1/2 of the time pressing the download button doesn't do anything for me. I might be imagining it but it could possibly be related to if there is a mixture of png and jpeg files? I was surprised to find out that many of the civitai jpegs have the prompt metadata that can be ready by automatic1111.

Is anyone else experiencing this?

Otherwise, when it works, it's great!

**Comment by bbdbby** [comment]
Also, why is this limited to only 10 images in the download? There are many creators who will put up more images than that. Seems like a strange limit.

**Comment by Mirabilis1729** [comment]
Yeah it seems the script appears to have stopped functioning. I've tried uninstalling/reinstalling etc to no avail. Pity, as it is super useful.

**Comment by bbdbby** [comment]
> 

Yeah, it seems to have completely stopped working for me. I'm using the latest version of Firefox, what browser are you using? Maybe it could be working in a different browser?

**Comment by Mirabilis1729** [comment]
> Yeah, it seems to have completely stopped working for me. I'm using the latest version of Firefox, what browser are you using? Maybe it could be working in a different browser?

I tried Brave and Vivaldi. No change. I suspect it is down to change at Civitai. We need to see if @tehrobber updates it.

**Comment by bbdbby** [comment]
In the meantime I have been using another Violentmonkey script called: Civitai网站辅助(English Version)
This version has been working for me and can even download up to 20 showcase images. Still wish it had a way to also download user images in batches, but it has been very helpful compared to nothing.
